Make installation instructions suggest installing a speech engine (#2061)
cursorless-dev/cursorless: docs/user/installation.md

# Installation

1. Install [Talon](https://talonvoice.com/)

>    - Make sure a speech engine is installed by clicking on the `Speech Recognition` tab after clicking the Talon tray icon

Reviewer: ```suggestion
   - Make sure a speech engine is installed (click the Talon tray icon, then select one from the `Speech Recognition` menu)
```
